lib/router/router.js

```diff
@@ -109,6 +126,17 @@ p.getInfo = function(req) {
     }
   }
 
+  // forward original image if no operation on image and browser support image format
+  if (routeInfo.imageSteps.length === 0 && (this.options.noop || routeInfo.urlInfo.query.noop)) {
```

Comment:
This option could be good only if someone would like to deactive your option to use webp when possible. Not sure if some would need it. It seems work fine.
The real problem to deal here how stop unnecessary operations when we get a request without operation parameters when save system resources is more important than webp conversion gain.
the format: "same" option would affect everything.
Actually I am pretty happy with my resized images use webp when possible. I just want to disable it on orginal to save resources.